docs/architecture/PROCESS.md

```diff
@@ -0,0 +1,68 @@
+# ADR Creation Process
+
+1. Copy the `adr-template.md` file. Use the following filename pattern: `adr-next_number-title.md`
+
+2. Create a draft Pull Request if you want to get an early feedback.
+
+3. Make sure the context and a solution is clear and well documented.
+
+4. Add an entry to a list in the [README](./README.md) file.
+
+5. Create a Pull Request to propose a new ADR.
+
+## ADR life cycle
+
+ADR creation is an **iterative** process. Instead of trying to solve all decisions in a single ADR pull request, we MUST firstly understand the problem and collect feedback through a GitHub Issue.
+
+1. Every proposal SHOULD start with a new GitHub Issue or be a result of existing Issues. The Issue should contain just a brief proposal summary.
```

Comment:
```suggestion
1. Every ADR proposal SHOULD start with a [new GitHub issue](https://github.com/tendermint/starport/issues/new/choose) or be a result of existing Issues. The Issue must contain a brief proposal summary.
```

Reply:
I would change this to replace "tendermint/starport" with "allinbits/cosmos-cash"